java/templates/deployment.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
 spec:
   securityContext:
     runAsUser: 1000
-    fsGroup: 2000
+    fsGroup: 1000
```

Comment:
does this actually work ... mine still opens as root even after Ive done this with the hmcts user

Reply:
it might need to be on the container level - can someone check this?

Reply:
No, this can be at pod level and it will be applied to all containers as far as I can tell:

https://kubernetes.io/docs/tasks/configure-pod-container/security-context/